newIDE/app/src/UI/PriceTag.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+
+import Text from './Text';
+import { makeStyles } from '@material-ui/core';
+
+type Props = {|
+  value: number,
+  /**
+   * To be used when the component is over an element for which
+   * we don't control the background (e.g. an image).
+   * This component should logically be invariant to selected theme
+   * when this option is activated
+   */
+  withOverlay?: boolean,
+|};
+
+const useStyles = makeStyles(theme => {
+  return {
+    container: {
+      borderRadius: 4,
+      padding: '2px 4px',
+      backdropFilter: props =>
+        props.withOverlay && theme.palette.type === 'light'
+          ? 'brightness(40%)'
+          : undefined,
+      backgroundColor: props =>
+        props.withOverlay && theme.palette.type === 'light'
+          ? undefined
+          : theme.palette.background.alternate.startsWith('#')
+          ? theme.palette.background.alternate + 'BB' // manually add opacity to the background hex color
+          : theme.palette.background.alternate,
+      color: props =>
+        props.withOverlay && theme.palette.type === 'light'
+          ? '#FAFAFA'
+          : undefined,
+    },
+  };
+});
+
+function PriceTag({ value, withOverlay }: Props) {
+  const classes = useStyles({ withOverlay });
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <div className={classes.container}>
+          <Text noMargin size="sub-title" color="inherit">
+            {i18n
+              .number(value / 100, {
+                minimumFractionDigits: 2,
+                maximumFractionDigits: 2,
+              })
+              .replace(/\D00$/, '')}
+            EUR
```

Comment:
I think the order is usually EUR 100. We could also use the symbol € => €10?

Reply:
This is for @LuniMoon ;)

Reply:
Agree that the opposite order is what I've been used to, but to check with Luni

Reply:
Sorry! that price got lost in translation hehe.

I've just bench a little here and there.

**Amazon**:
<img width="276" alt="Screenshot 2022-10-04 at 11 00 37" src="https://user-images.githubusercontent.com/103995399/193779085-5b67c369-555b-4293-a698-fa19c1ad7b7d.png">

**Google Search** results:
<img width="693" alt="Screenshot 2022-10-04 at 11 03 31" src="https://user-images.githubusercontent.com/103995399/193779920-1230a846-4958-4bb5-ac63-3565ccbf7da0.png">

**United Nude** (shoes):
<img width="197" alt="Screenshot 2022-10-04 at 11 02 40" src="https://user-images.githubusercontent.com/103995399/193779713-de2d1922-0a4e-40cc-af03-32d94b76d240.png">

**Quicksilver** (clothes):
<img width="249" alt="Screenshot 2022-10-04 at 11 03 11" src="https://user-images.githubusercontent.com/103995399/193779825-a7c7dabd-1242-4857-8d60-613b6db6a141.png">

**Vans** (clothes):
<img width="225" alt="Screenshot 2022-10-04 at 11 04 13" src="https://user-images.githubusercontent.com/103995399/193780068-b530619d-104a-4497-8f61-6bbc1d457df9.png">

**KLM** (flights)
<img width="258" alt="Screenshot 2022-10-04 at 11 05 01" src="https://user-images.githubusercontent.com/103995399/193780209-ae98b732-20d2-486f-8aff-e1c7b67119a8.png">

**I love NY** official store:
<img width="216" alt="Screenshot 2022-10-04 at 11 10 49" src="https://user-images.githubusercontent.com/103995399/193781377-9f39443e-f79a-4355-86af-911987331b2a.png">


So, let's go with the majority, and **keep the "€" at the beginning of the price**.
As for the decimals, we'll use 2 (if needed), and I personally have a preference for a dot separating the cents from the units (I'm from the american school), but I dunno if that's good practice in Europe or the rest of the world (?).
I do see sites using commas and dots interchangeably...

Reply:
Note on the side: In my context, **commas are used for big numbers like "2,599" and dots for cents "2.59"**.

But I do know that depending on the language, numbers are written differently (each language has certains rules for using commas and dots, and sometimes they vary from each other... I do have some books as reference if you wanna read about it lol).

Reply:
> I personally have a preference for a dot separating the cents from the units

This should be handled by the native internationalization library.
In French, this is really weird for non-english speaking people to see a price like `1,255.45`. I guess other countries have the same kind of issues.